src/main/kotlin/me/modmuss50/mpp/HttpUtils.kt

```diff
@@ -26,6 +26,14 @@ class HttpUtils(val exceptionFactory: HttpExceptionFactory = DefaultHttpExceptio
         .build()
     val json = Json { ignoreUnknownKeys = true }
 
+    inline fun <reified T> get(url: String): T {
+        return request(
+            Request.Builder()
+                .url(url),
+            emptyMap()
+        )
+    }
```

Comment:
This isnt needed, there is already a `get`. Just pass the same headers as used when publishing a mod.